### ISSUE TITLE
numba.errors.RedefinedError: ... (step: analyzing bytecode) $const_0.5

### ISSUE THREAD

**Issue by slayoo** [numba/numba#5413]
## Reporting a bug

Using Numba 0.48.0 with Python 3.7.6, running the script below:

```python
import numba

@numba.njit()
def fun(a, b, c, d):
    return 0

@numba.njit()
def test():
    a = (0, 1)
    return fun(*a, +.5, 0) * fun(*a, 0, 0) + fun(*a, +.25, 0) * fun(*a, 0, 0)

if __name__ == '__main__':
    test()
```

results in the following error:

```
$ python3  bug.py
Traceback (most recent call last):
  File "bug.py", line 13, in <module>
    test()
  File "/usr/local/lib/python3.7/dist-packages/numba/dispatcher.py", line 409, in _compile_for_args
    error_rewrite(e, 'interpreter')
  File "/usr/local/lib/python3.7/dist-packages/numba/dispatcher.py", line 344, in error_rewrite
    reraise(type(e), e, None)
  File "/usr/local/lib/python3.7/dist-packages/numba/six.py", line 668, in reraise
    raise value.with_traceback(tb)
numba.errors.RedefinedError: Failed in nopython mode pipeline (step: analyzing bytecode)
$const_0.5
```

**Comment by slayoo** [comment]
For the record, even shorter code to reproduce it:

```python
import numba
@numba.njit(numba.float64())
def test():
    a = (0, 1)
    return fun(*a, +.5, 0) * fun(*a, 0, 0) + fun(*a, +.25, 0) * fun(*a, 0, 0)
```

**Comment by stuartarchibald** [comment]
Thanks for the report. Unfortunately this is a known bug, root cause is the same as https://github.com/numba/numba/issues/5243, PR https://github.com/numba/numba/pull/5351 fixes it and is scheduled for 0.49.0RC which will (hopefully) be generally available later this week.

**Comment by stuartarchibald** [comment]
Fixed by #5351, closing. Version 0.49.0* will ship with this.
